<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnaSayfa</class>
 <widget class="QMainWindow" name="AnaSayfa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ana Sayfa</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>10</y>
      <width>561</width>
      <height>581</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="spacing">
      <number>5</number>
     </property>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLabel" name="label_4">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="label_6">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_5">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="label_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>100</y>
      <width>200</width>
      <height>400</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="open_image">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>50</y>
       <width>170</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Open Image File</string>
     </property>
    </widget>
    <widget class="QPushButton" name="desired_image">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>110</y>
       <width>170</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Open Desired Image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="histogram">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>170</y>
       <width>170</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Histogram</string>
     </property>
    </widget>
    <widget class="QPushButton" name="equalization">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>230</y>
       <width>170</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Equalization</string>
     </property>
    </widget>
    <widget class="QPushButton" name="specification">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>290</y>
       <width>170</width>
       <height>50</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Times New Roman</family>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Specification</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>open_image</sender>
   <signal>clicked()</signal>
   <receiver>AnaSayfa</receiver>
   <slot>RESIM_AC()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>104</x>
     <y>171</y>
    </hint>
    <hint type="destinationlabel">
     <x>136</x>
     <y>26</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>desired_image</sender>
   <signal>clicked()</signal>
   <receiver>AnaSayfa</receiver>
   <slot>RESIM_AC2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>154</x>
     <y>241</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>62</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>histogram</sender>
   <signal>clicked()</signal>
   <receiver>AnaSayfa</receiver>
   <slot>HISTOGRAM()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>43</x>
     <y>307</y>
    </hint>
    <hint type="destinationlabel">
     <x>59</x>
     <y>515</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>equalization</sender>
   <signal>clicked()</signal>
   <receiver>AnaSayfa</receiver>
   <slot>EQUALIZATION()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>56</x>
     <y>363</y>
    </hint>
    <hint type="destinationlabel">
     <x>88</x>
     <y>543</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>specification</sender>
   <signal>clicked()</signal>
   <receiver>AnaSayfa</receiver>
   <slot>SPECIFICATION()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>561</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>RESIM_AC()</slot>
  <slot>RESIM_AC2()</slot>
  <slot>HISTOGRAM()</slot>
  <slot>EQUALIZATION()</slot>
  <slot>SPECIFICATION()</slot>
 </slots>
</ui>
